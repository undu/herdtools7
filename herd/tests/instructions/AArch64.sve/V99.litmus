AArch64 V01
(* XOR vectors (unpredicated) *)
{
}
P0                    ;
 MOV Z0.S, #191       ;
 MOV Z1.S, #64        ;
 EOR Z2.S,Z1.S,Z0.S   ;
 MOV Z3.D, #3         ;
 MOV Z4.D, #4         ;
 EOR Z5.D,Z4.D,Z3.D   ;

forall 0:V2.4S={255,255,255,255} /\ 0:V5.4S={7,0,7,0}
