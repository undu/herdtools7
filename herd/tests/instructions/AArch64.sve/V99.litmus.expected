Test V01 Required
States 1
0:V2.4S={255,255,255,255}; 0:V5.4S={7,0,7,0};
Ok
Witnesses
Positive: 1 Negative: 0
Flag Scalable-Vector-Extensions-is-work-in-progress
Condition forall (0:V2.4S={255,255,255,255} /\ 0:V5.4S={7,0,7,0})
Observation V01 Always 1 0
Hash=3a251b4279be1b01de64a29c16ed7a9e

